package.json

```diff
@@ -6,6 +6,7 @@
     "lodash": "^4.17.5",
     "react": "^16.2.0",
     "react-dom": "^16.2.0",
+    "react-router": "^4.2.0",
```

Comment:
had to add this to make ```yarn start``` work?

Reply:
weird, but no problem we'll need it when we add the stat calculator and stuff